editor/src/clj/editor/debug_view.clj

```diff
@@ -473,38 +474,39 @@
 
 (def ^:private mobdebug-port 8172)
 
-(defn- show-connect-failed-dialog! [target-address port ^Exception exception]
-  (let [msg (str "Failed to attach debugger to " target-address ":" port ".\n"
-                 "Check that the game is running and is reachable over the network.\n")]
-    (log/error :msg msg :exception exception)
-    (dialogs/make-info-dialog
-      {:title "Attach Debugger Failed"
-       :icon :icon/triangle-error
-       :header msg
-       :content (.getMessage exception)})))
+(defn- show-connect-failed-info! [target-address port ^Exception exception workspace]
+  (ui/run-later
+    (let [msg (str "Failed to attach debugger to " target-address ":" port ".\n"
+                   "Check that the game is running and is reachable over the network.\n"
+                   "Error message:\n" (.getMessage exception))]
+      (log/error :msg msg :exception exception)
+      (notifications/show!
```

Comment:
notification instead of dialog . it looks like this: 
![CleanShot 2024-08-07 at 09 12 11@2x](https://github.com/user-attachments/assets/aca207c1-03b3-43b0-a9b3-51313b8dbce1)

Reply:
The double error messages seem a bit reduntant. Maybe we should just show the text from the error instead of "Failed to attach to debugger ...", followed by a newline and the helpful info text?

So it would become:
```
Failed to connect to debugger on localhost:812

Check that the game is running and is reachable over
the network.
```

Reply:
Here is fixed version
![CleanShot 2024-08-07 at 10 54 30@2x](https://github.com/user-attachments/assets/da155a49-6734-46f5-a611-9eab0f6f360a)